include/mbedtls/mbedtls_config.h

```diff
@@ -2533,17 +2533,40 @@
  */
 #define MBEDTLS_ECJPAKE_C
 
+/**
+ * \def MBEDTLS_ECP_LIGHT
+ *
+ * Enable part of the elliptic curve over GF(p) library.
+ *
+ * \warning This option is experimental and its scope may change in future
+ * versions. Most users will want to enable the stable option #MBEDTLS_ECP_C
+ * instead. Enabling this option rather than #MBEDTLS_ECP_C is mostly useful to
+ * minimize code size when you need none of #MBEDTLS_ECDH_C, #MBEDTLS_ECDSA_C,
+ * or #MBEDTLS_ECJPAKE_C (for example because you have PSA drivers for them) but
+ * still want support for (un)parsing ECC keys in the PK module.
+ *
+ * \note Currently, this option enables the following parts of ecp.c: (un)parsing
+ * and validation of keys, and info functions. It excludes scalar multiplication
+ * and everything that depends on it, such as keypair generation.
+ *
+ * Module:  library/ecp.c
+ *
+ * Requires: MBEDTLS_BIGNUM_C and at least one MBEDTLS_ECP_DP_XXX_ENABLED
+ */
+#define MBEDTLS_ECP_LIGHT
```

Comment:
Per my reasoning in https://github.com/Mbed-TLS/mbedtls/issues/6839#issuecomment-1506571727, I don't think we should make `MBEDTLS_ECP_LIGHT` public, not even as an experimental feature.